### ISSUE TITLE
wallet conflicts prevent me from spending totally valid outputs

### ISSUE THREAD

**Issue by 1Hyena** [bitcoinxt/bitcoinxt#275]
![bitcoinxt_fuck](https://user-images.githubusercontent.com/3479475/32134698-8001aed4-bbfb-11e7-8e73-c76d3d425be2.png)

I am unable to spend anything from my wallet with v0.11G2 because there are a couple of really old conflicted TXs or something, I don't know. Fact is that all block explorers tell me a valid balance on my address but bitcoinxt is unable to spend anything. very fustrating, even -rescan did not fix it, now I tried with -zapwallet mode 2 and rescanning again, hopefully this works

speaking of bitcoin cash obviously

**Comment by 1Hyena** [comment]
yep -zapwallet mode 2 did not work either! there are no more conflicting TXs in my TX history but I still can't spend anything. this is ridiculous

**Comment by 1Hyena** [comment]
Just letting you know that I was able to manually construct the needed TXs from the debug window using RPCs. I then pushed the raw transaction hex to the network using the web interface from viabtc mining pool. so it is definitely some wallet software related bug, nothing wrong with my transactions

**Comment by dgenr8** [comment]
@1Hyena I don't know how to go further without asking for a copy of your wallet which I'm not optimistic will be acceptable for you...

**Comment by 1Hyena** [comment]
@dgenr8 yes :D this is true. well since I created this issue I actually soon had to change my wallet to bitcoin unlimited because xt prevented me from creating TXs due to named issue. I can only say that BUCash wallet also shows me that some of my TXs are unconfirmed since 10. 28. 2017 but at least it lets me spend my UTXOs. unfortunately I had to make another switch from bitcoinxt to BUCash because xt node stopped receiving new blocks, even though I reconnected the VPN and deleted the mempool.dat and even tried without VPN at all, it simply failed to get any blocks. I have one full-time xt node remaining though, hopefully I won't have to switch that too because I actually like XT's philosophy regarding double-spend relaying.
